content/blog/Installation.md

```diff
@@ -1,6 +1,113 @@
 ---
-date: 2024-02-20
-title: Installing orchestrator using helm chart on OpenShift
+date: 2024-03-13
+title: Installing the Orchestrator on existing RHDH instance
 ---
 
-{{< remoteMD "https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/README.md?raw=true" >}}
+When RHDH is already installed and used, it is required to avoid re-installing it by the Helm chart.
+There are few steps required add the Orchestrator to such environment:
+1. Install the backing services by the Helm chart by disabling the RHDH installation
+2. Manually patching the existing RHDH ConfigMap resources with the Orchestrator plugin configuration.
+3. Import the Orchestrator software templates into Backstage catalog
```

Comment:
Do you think it makes sense to refer to this points below?

Reply:
it is mentioned at the bottom of the document.